src/demo_central.cpp

```diff
@@ -86,6 +86,7 @@ class Arm {
   }
 
   bool moveHome() {
+    //setGoalIntermediate();
```

Comment:
This is commented out here (and 3 others places in the file).  I would remove these lines.